CHANGELOG.rst

```diff
@@ -1,6 +1,13 @@
 django-s3-storage changelog
 ===========================
 
+0.9.9
+-----
+
+- Added settings for disabling gzip compression (@leonsmith, @bartvds)
+- Bug fix for relative upload paths (@leonsmith)
```

Comment:
Yes, let's do a release! :D

Could you add:

``` rst
- Bug fix for detecting empty directories (@etianen).
- Automatic conversion of windows path separators on upload (@etianen).
```

Also, the general policy here is that credit for features goes to the implementors, not the requester, unless they make a noticeable contribution towards the design. So if you could also remove bartvds from the the credits, that would be good.

(Not trying to be harsh, as requesting features and reporting bugs *is* important, but actually implementing stuff is where the real effort lies, and this is how it's been done to this point)